### ISSUE TITLE
how to contribute?

### ISSUE THREAD

**Issue by ZhengHe-MD** [Da-Kuang/react-custom-table#1]
不清楚你是怎么调试代码的？希望在README里面能有how to contribute的说明。

**Comment by ZhengHe-MD** [comment]
demo是可以跑起来的。代码我差不多也过了一遍。不过要玩起来还是有点费劲， 能不能简单介绍一下你开发时的workflow。谢谢

**Comment by Da-Kuang** [comment]
yeah, sorry about that, the start script doesn't make sense for now,  since there is no index.html in the root, i will remove it. For now, you can just `npm run demo`, it will start a dev server over the demo folder, it will load the code.  And I will add hot reload soon. Thanks
